control-plane/README.md

```diff
@@ -27,12 +27,12 @@ The job control plane is used for controlling and monitoring the jobs.
 2. Log into Firebase CLI using "firebase login"
 3. Run "firebase use --add" to add an alias. When prompted, select your **Project ID**, then give your Firebase project an alias. For more info, visit this [link](https://firebase.google.com/docs/cli#add_alias).
 4. Finally, run ```yarn deploy``` to deploy to Firebase Hosting. When you run the command, Firebase CLI should give a link to your web app.
-5. Update .github/workflows with your GCP-Project ID for automatic updates
+5. Update **.github/workflows** with your **GCP-Project ID** for automatic updates
```

Comment:
What is this for? 
Everything in .github/workflows is for repository config on Github specifically. This should not be part of the instructions.

Reply:
just delete all the files in `.github/workflows` that `firebase init` generated. We don't need those for this repo.